src/rTerminal.ts

```diff
@@ -247,12 +247,24 @@ export async function runTextInTerm(text: string): Promise<void> {
             // Surround with ANSI control characters for bracketed paste mode
             text = `\x1b[200~${text}\x1b[201~`;
         }
-        term.sendText(text);
+        term.sendText(text, execute);
     } else {
         const rtermSendDelay: number = config().get('rtermSendDelay');
-        for (const line of text.split('\n')) {
-            await delay(rtermSendDelay); // Increase delay if RTerm can't handle speed.
-            term.sendText(line);
+        const split = text.split('\n');
+        const n_splits = split.length;
+        let count = 0;
+        for (const line of split) {
```

Comment:
I think a couple of lines could be saved here using `.entries()`:

```typescript
for (const [count, line] of split.entries()) {
```

Reply:
Much nicer indeed.